src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1360,6 +1363,26 @@ MavlinkReceiver::handle_message_battery_status(mavlink_message_t *msg)
 	battery_status.cell_count = cell_count;
 	battery_status.connected = true;
 
+	// Get the battery level thresholds.
+	float bat_emergen_thr;
+	float bat_crit_thr;
+	float bat_low_thr;
+	param_get(_p_bat_emergen_thr, &bat_emergen_thr);
+	param_get(_p_bat_crit_thr, &bat_crit_thr);
+	param_get(_p_bat_low_thr, &bat_low_thr);
```

Comment:
Yes, I bench tested the software by feeding in a MAVLink BATTERY_STATUS message with a varying "battery_status.remaining" value.  The software annunciated the different battery conditions (e.g. Critical Battery) as expected.